lib/client/weblogin.go

```diff
@@ -225,6 +226,26 @@ type SSHLoginMFA struct {
 	PreferOTP bool
 }
 
+// SSHLoginPasswordless contains SSH login parameters for passwordless login.
+type SSHLoginPasswordless struct {
+	SSHLogin
+	Stderr io.Writer
```

Comment:
Maybe:

```suggestion
	StderrOverride io.Writer
```

then document and default to os.Stderr if nil?